### PR TITLE
Continue new design

### PR DESCRIPTION
**Pull request by ruveyda-kisla**
Added modal
Added Disclosure
Added successful transaction page
Added step four of payment 
Added Cancel order modal
Added Expired order modal
Added payment details modal (this needs help)

the data is for test and the structure of the components will change upon changing back end response
to test step four I changed the state of steps to 4 
`const [step, setStep] = useState(4);`

to test success modal I changed the layout in App and created a temporary constant `isSuccessful` to show it, to see the modal you need to make that boolean true right now its false. ` const isSuccessful = false;`

remember, this version of codes still need maintenance and they are not finished, we are pended on back end new data and responsive.

### CODE REVIEW

File: src/components/CConnectWallet/index.tsx
Comment (ceydae) [inline src/components/CConnectWallet/index.tsx:1]: remove file

File: src/components/CDisclosure/index.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ interface CDisclosureProps {
   title: string;
   subTitle?: string;
   content: React.ReactNode | string;
-  icon?: React.ReactNode | string;
+  icon?: string | undefined;
```
Comment (ceydae) [inline src/components/CDisclosure/index.tsx]: remove undefined where u already have ?

File: src/components/Modals/cancelOrderModal/index.tsx
Comment (ceydae) [inline src/components/Modals/cancelOrderModal/index.tsx:1]: capitalize the first letter Cancel

**Review by ceydae — APPROVED** [review]

**Comment by ceydae** [comment]
Keep my order has no hover effect.

**Comment by ceydae** [comment]
Token is  shown like this 
Screenshot 2024-11-22 at 15 44 25

**Comment by ceydae** [comment]
see in explorer => See in explorer.

Also needs a hover effect

**Comment by ceydae** [comment]
Download script hover effect

**Comment by ceydae** [comment]
Cancel Order and More info buttons stay when the "Transaction Info" modal is opened. They should also be blurred.

**Comment by ceydae** [comment]
Need Help should be linked to https://docs.wagent.app and should have a hover effect.

**Comment by ceydae** [comment]
In `App.tsx`, change isExpired from this:

```
  const isExpired = calculateRemainingTime(data?.expiredTimestamp as number) ? true : false;
```

to this:

```
  const isExpired = !!calculateRemainingTime(data?.expiredTimestamp as number);
```

And add a blank line. Also fix the import ordering in the file

**Comment by ceydae** [comment]
I like the responsive model very much.
The only thing that can be better i think is the delay when a modal is about to open. I think it's unnecessary and too much.

**Comment by ruveyda-kisla** [comment]
> Keep my order has no hover effect.

Done

**Comment by ruveyda-kisla** [comment]
> Cancel Order and More info buttons stay when the "Transaction Info" modal is opened. They should also be blurred.

for this I need help

**Review by dousdogukan — APPROVED** [review]

**Review by ceydae — APPROVED** [review]

**Pull request merged**